utils/line-directive

```diff
@@ -262,13 +271,19 @@ def run():
     glitch in [fox.box:28 assert six
     EOF
     >>> print(subprocess.check_output([sys.executable, __file__, 'foo.bar',
-    ...                                '21', target1.name]).rstrip())
+    ...                                '21', target1.name],
+    ...                               universal_newlines=True))
     5
+    <BLANKLINE>
```

Comment:
Found a better fix. This is to add `end=''` to the print statement, avoiding the blankline added by print